src/components/DateRangePickerInput.jsx

```diff
@@ -45,6 +45,7 @@ const propTypes = forbidExtraProps({
   readOnly: PropTypes.bool,
   showCaret: PropTypes.bool,
   showDefaultInputIcon: PropTypes.bool,
+  inputIconPosition: PropTypes.oneOf('before', 'after'),
```

Comment:
There's a couple alternatives here; one is importing the shape but only extracting the one propType - another is putting this propType in its own file, and importing that in all the places it's needed. I'd prefer that approach tbh :-)